tests/distributed/test_dist_graph_store.py

```diff
@@ -203,15 +205,16 @@ def check_server_client(shared_mem):
     # We cannot run multiple servers and clients on the same machine.
     serv_ps = []
     ctx = mp.get_context('spawn')
-    for serv_id in range(1):
-        p = ctx.Process(target=run_server, args=(graph_name, serv_id, 1, shared_mem))
+    for serv_id in range(num_servers):
+        p = ctx.Process(target=run_server, args=(graph_name, serv_id,
+                                                 num_clients, shared_mem))
         serv_ps.append(p)
         p.start()
 
     cli_ps = []
-    for cli_id in range(1):
+    for cli_id in range(num_clients):
         print('start client', cli_id)
-        p = ctx.Process(target=run_client, args=(graph_name, cli_id, g.number_of_nodes(),
+        p = ctx.Process(target=run_client, args=(graph_name, 0, num_clients, g.number_of_nodes(),
```

Comment:
why args=(graph_name, 0, num_clients, g.number_of_nodes(), 
not args=(graph_name, cli_id, g.number_of_nodes()

Reply:
0 is partition id. partition id and client id are different. one machine has only one partition, but one machine can run multiple servers and multiple clients.